js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
We keep going back and forth.

It's pretty important to have the latest info available when making a decision on which moderator to choose.

On the other hand, that's how I used to have it, and I forget who, but someone was adamant faster was more important that accurate.

@hoffmabc you want to weigh in here?